config/default.yml

```diff
@@ -0,0 +1,11 @@
+toolkit:
+  project_id: project-id
+  clone:
+    asda_url: 'https://webgate.ec.europa.eu/fpfis/files-for/automate_dumps'
```

Comment:
`asda-url` instead of `asda_url`

Reply:
This is again a configuration vs command options case, so config is `asda_url` and command option is `asda-url`. I know it's wired but yeah, for the sake of consistency...

Reply:
done.